deploys/main.tf

```diff
@@ -0,0 +1,130 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+provider "sym" {
+  org = var.sym_org_slug
+}
+
+
+resource "sym_flow" "this" {
+  name  = "ci-approval"
+  label = "CI Approval"
+
+  template       = "sym:template:approval:1.0.0"
+  implementation = "${path.module}/impl.py"
+  environment_id = sym_environment.this.id
+
+  vars = var.flow_variables
+
+  params = {
+    # allowed_sources_json defines the sources from which this flow can be
+    # invoked from. Valid values: "api", "slack".
+    allowed_sources_json = jsonencode(["api"])
+
+    # prompt_fields_json defines custom form fields for the Slack modal that
+    # requesters fill out to make their requests.
+    prompt_fields_json = jsonencode([
+      {
+        name     = "workflow_url"
```

Comment:
We define the prompt_field to be `workflow_url` here, but we rely on `workflow_id` in `impl.py`